packages/xo-server/src/xo-mixins/resource-sets.mjs

```diff
@@ -55,10 +55,10 @@ const normalize = set => ({
   limits: set.limits
     ? map(set.limits, limit =>
         isObject(limit)
-          ? limit
+          ? { ...limit, usage: limit.usage ?? 0 }
```

Comment:
IMHO, we could simplify a lot the code by considering than no usage mean `0`.

What do you think?

Reply:
This added line was just to avoid using `... || 0` every time we need to perform an operation on `limit.usage`.
`limit.usage += quantity` will return `NaN` in case we have not added this line.

Reply:
Makes sense :+1: